examples/cloud_run_cloudevents/main.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sample creates a function that accepts a Cloud Event per
+# https://github.com/cloudevents/sdk-python
+import sys
+
+
+def hello(cloudevent):
+    print(
+        f"Received event with ID: {cloudevent['id']}",
+        file=sys.stdout,
+        flush=True
+    )
```

Comment:
```suggestion
    print(f"Received event with ID: {cloudevent['id']}")
```
The Dockerfile should set `ENV PYTHONUNBUFFERED TRUE` instead (this should probably be set for all Dockerfile examples in this repo as well).

Reply:
Set env var in all sample dockerfiles